src/utils/PomodoroTimer.js

```diff
@@ -5,20 +5,32 @@ export const PomodoroTimer = {
     seconds = seconds || 0;
     seconds = seconds + minutes * 60;
 
+    const immutableseconds = seconds
+
+    let end = new Date()
+    end.setSeconds(end.getSeconds() + immutableseconds)
```

Comment:
I don't know if we need `immutableseconds`. Perhaps this is sufficient:

```suggestion
    end.setSeconds(end.getSeconds() + seconds)
```

Reply:
done